src/olympia/stats/models.py

```diff
@@ -55,7 +52,11 @@ class UpdateCount(StatsSearchMixin, models.Model):
 
     class Meta:
         db_table = 'update_counts'
-
-        # Additional indices on this table (on dev, stage and prod):
-        # * KEY `addon_and_count` (`addon_id`,`count`)
-        # * KEY `addon_date_idx` (`addon_id`,`date`)
+        indexes = [
+            # FIXME: some of these might redundant.
```

Comment:
Same here, maybe link to https://github.com/mozilla/addons-server/issues/5712?

Reply:
1163b56517f2961ae5f136dc6d937d276fcfc897